sdk/python/tests/performance_tests.py

```diff
@@ -205,91 +224,111 @@ def get_client() -> TektonClient:
     return client
 
 
-def load_pipeline_functions() -> [(Callable, str)]:
-    pipeline_functions = []
+def get_project_root_dir() -> Path:
+
+    script_path_presumed = "sdk/python/tests/performance_tests.py"
+    script_path_actually = Path(__file__)
+    project_root_folder = script_path_actually.parents[3]
+
+    assert script_path_actually == project_root_folder.joinpath(script_path_presumed), \
+        "Can not determine project root folder. Was this script file moved or renamed?"
+
+    return project_root_folder
+
+
+def load_test_config() -> dict:
 
-    from testdata.sequential import sequential_pipeline
```

Comment:
the diff shown here is a little misleading, GH tries to align lines of code that are not related